src/test/groovy/org/mockito/release/internal/gradle/VersioningPluginTest.groovy

```diff
@@ -0,0 +1,61 @@
+package org.mockito.release.internal.gradle
+
+import org.gradle.testfixtures.ProjectBuilder
+import org.junit.Rule
+import org.junit.rules.TemporaryFolder
+import org.mockito.release.version.VersionInfo
+import spock.lang.Specification
+
+class VersioningPluginTest extends Specification {
+
+    @Rule TemporaryFolder tmp = new TemporaryFolder()
+    def project
+    def initialVersionFileContent = "version=1.0.0\nnotableVersions=0.1.0\n"
```

Comment:
I suggest to inline this variable. It's nice to avoid duplication. Problem is that tests are easier to read and maintain if they have a little bit of conscious duplication. DAMP VS DRY principle for unit tests :)